File: Count Good Triplets.go
package main

func countGoodTriplets(arr []int, a int, b int, c int) int {
	ans := 0
	for i := 0; i < len(arr); i++ {
		for j := i + 1; j < len(arr); j++ {
			for k := j + 1; k < len(arr); k++ {
				if float64(abs(arr[i]-arr[j])) <= float64(a) &&
					float64(abs(arr[j]-arr[k])) <= float64(b) &&
					float64(abs(arr[i]-arr[k])) <= float64(c) {
					ans++
				}
			}
		}
	}
	return ans
}

func abs(x int) int {
	if x < 0 {
		return -x
	}
	return x
}

File: Count Good Triplets.js
function countGoodTriplets(arr, a, b, c){
    //initialize answer
    let ans = 0;
    //for loop to check 0 <= i < j < k < arr.length |arr[i] - arr[j]| <= a |arr[j] - arr[k]| <= b |arr[i] - arr[k]| <= c
    for (let i = 0; i < arr.length; i++){
        for (let j = i + 1; j < arr.length; j++){
            for (let k = j + 1; k < arr.length; k++){
                if (Math.abs(arr[i] - arr[j]) <= a && Math.abs(arr[j] - arr[k]) <= b && Math.abs(arr[i] - arr[k]) <=c){
                    ans += 1;
                }
            }
        }
    }
    return ans;
}